api/server.js

```diff
@@ -501,6 +501,11 @@ async function main(){
         res.status(200).json(await getUserGradesAsFraction(apiAuthClient, req.body.email));
     });
 
+    // Responds with the grade projections for the specified student
+    app.get('/api/admin/getStudentProjection', async (req, res) => {
```

Comment:
`get` in the route is repetitive.  The only way we can trigger this endpoint is with a get request.